equinox/internal/_loop/common.py

```diff
@@ -61,6 +61,17 @@ class _BufferItem(Module):
     def set(self, x, *, pred=True):
         return self._buffer._set(pred, self._item, x)
 
+    def add(self, x, *, pred=True):
+        return self._buffer._set(pred, self._item, self._buffer[self._item] + x)
+
+    def multiply(self, x, *, pred=True):
+        return self._buffer._set(pred, self._item, self._buffer[self._item] * x)
+
+    def divide(self, x, *, pred=True):
+        return self._buffer._set(pred, self._item, self._buffer[self._item] / x)
+
+    def power(self, x, *, pred=True):
+        return self._buffer._set(pred, self._item, self._buffer[self._item] ** x)
```

Comment:
Let's make it so that e.g. `add` lowers to `array.at[...].add`.

Probably replace `_set(self, pred, item, x) -> _op(self, pred, item, x , op)`, where `op` is a string indicating which method to call: `getattr(array.at[...], op)`.